main.js

```diff
@@ -0,0 +1,15 @@
+const p = document.querySelector("#clickcount");
+const storage = window.localStorage;
+
+let i = storage.getItem("clickcount");
+if(!i) {
+	i = 0;
+	storage.setItem("clickcount", 0);
+}
+
+p.innerHTML = i;
```

Comment:
you duplicated this row. Better will create a function for increment `i` and for render `i`.